client/api/InnerClient.cs

```diff
@@ -176,6 +190,12 @@ public void OnPollError(string message)
         {
 
         }
+
+        public void OnPollCompleted(IList<string> identifiers)
```

Comment:
```suggestion
        public void OnPollCompleted(IList<string> identifiers) =>
            OnNotifyFlagsLoaded(identifiers);
```

Reply:
I like the change, and committed it, but subsequently reverted. It't just it doesn't match the style here for other one liner methods.

Reply:
yes, defintiely not suggesting to change only one; I'd do them all
and whack all the Log.Isenabled checks
but it's your code and call - it's only a suggestion!
(I am alergic to braces as F# is my daily driver and they don't let you put in redundant noise like that :D)